### ISSUE TITLE
Wrong mapping of ASCII coding

### ISSUE THREAD

**Issue by chouex** [farhadi/node-smpp#26]
i try decode char: 0x40 with data_coding: 0 which is the '@' in ASCII table, but decode to '¡' when using encodings['ASCII'].decode()

i found ASCII decode using this table 
```
encodings.ASCII = { // GSM 03.38
	chars: '@£$¥èéùìòÇ\nØø\rÅåΔ_ΦΓΛΩΠΨΣΘΞ\x1BÆæßÉ !"#¤%&\'()*+,-./0123456789:;<=>?¡ABCDEFGHIJKLMNOPQRSTUVWXYZÄÖÑÜ§¿abcdefghijklmnopqrstuvwxyzäöñüà',
```

is it decoded ASCII data using GSM 03.38 table?

**Comment by chouex** [comment]
may be your code is right, data_coding: 0 is SMSC Default Alphabet (SMPP 3.4) / MC Specific (SMPP 5.0)
http://www.developershome.com/sms/gsmAlphabet.asp

https://en.wikipedia.org/wiki/Short_Message_Peer-to-Peer

maybe you need to change the name from ASCII to GSM0338

**Comment by chouex** [comment]
i use this correct encoding
```
pdu.short_message.message=defs.encodings.LATIN1.decode(defs.encodings.ASCII.encode(pdu.short_message.message));
```

**Comment by hkulekci** [comment]
I guess, something wrong in https://github.com/farhadi/node-smpp/blob/master/lib/defs.js#L1201-L1202 this line. What do you think? 

Why IA5 and ASCII data encoding is the same? Encoding dont not work correctly on server side implementation? I changed the codes following and test on server side. I works but I am using my own GSM-7 encoder and decoder if data encoding is 1. @farhadi What is that mean? 

```
	ENCODING: {
		ASCII:              0x00, 
		IA5:                  0x01,
		LATIN1:           0x03,
```

**Comment by chouex** [comment]
i think it because data_coding: 0x00 ban be both ASCII and GSM 03.38, this library see 0x00 as GSM 03.38 while my smpp server see 0x00 as ASCII.

may be user should specify which encoding is when data_coding= 0x00

**Comment by hkulekci** [comment]
Hi @chouex , What is change if we change this value as 0x00? Is something went wrong? I tested on 0x00, apperently, nothing changed.

**Comment by farhadi** [comment]
@chouex I fixed this issue at 68288cb8ef31f012a2d63ec6d9ace6c50b2b7857
Now you can change default encoding for `data_coding:0` like this:
```
smpp.encodings.default = 'LATIN1';
```
@hkulekci I took encoding values from the spec (4.7.7 data_coding), duplicates are just alternative names.

**Comment by hkulekci** [comment]
Thank you @farhadi . I understand that, in fact, IA5 and ASCII has same code on smpp.
